src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/visitbalances/VisitBalanceMappingApiService.kt

```diff
@@ -29,7 +29,7 @@ class VisitBalanceMappingApiService(@Qualifier("mappingApiWebClient") webClient:
     .awaitFirstOrDefault(CreateMappingResult())
 
   suspend fun getByNomisVisitBalanceAdjustmentIdOrNull(nomisVisitBalanceAdjustmentId: Long): VisitBalanceAdjustmentMappingDto? = adjustmentApi
-    .prepare(adjustmentApi.getMappingByNomisId2RequestConfig(nomisVisitBalanceAdjustmentId))
+    .prepare(adjustmentApi.getMappingByNomisId1RequestConfig(nomisVisitBalanceAdjustmentId))
```

Comment:
this is a bit naff - we should really choose unique names.  When there are conflicts swagger adds a `1` and a `2` etc. In this case `1` was then removed from the mapping PR which then meant the numbers for the others decreased by one.